### ISSUE TITLE
[master] export translation: TypeError: 'unicode' does not have the buffer interface

### ISSUE THREAD

**Issue by moylop260** [odoo/odoo#18012]
Impacted versions:
master

Steps to reproduce:
1. Go menu Configuration -> Translations -> Export Translation
2. Choose language: Spanish MX, Format file: PO File, Apps to export: Sale Management

Current behavior:
```bash
  File "master/addons/web/controllers/main.py", line 914, in call_button
    action = self._call_kw(model, method, args, {})
  File "master/addons/web/controllers/main.py", line 902, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "master/odoo/odoo/api.py", line 681, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "master/odoo/odoo/api.py", line 672, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "master/odoo/addons/base/module/wizard/base_export_language.py", line 38, in act_getfile
    tools.trans_export(lang, mods, buf, this.format, self._cr)
  File "master/odoo/odoo/tools/translate.py", line 710, in trans_export
    _process(format, modules, translations, buffer, lang)
  File "master/odoo/odoo/tools/translate.py", line 666, in _process
    writer.write_infos(modules)
  File "master/odoo/odoo/tools/translate.py", line 618, in write_infos
    'now': datetime.utcnow().strftime('%Y-%m-%d %H:%M')+"+0000",
TypeError: 'unicode' does not have the buffer interface
```

Expected behavior:
Exported

**Comment by Yenthe666** [comment]
Hi @moylop260,

Thank you for your report.
This is indeed a bug (and for all langs 😉):
![image](https://user-images.githubusercontent.com/6352350/27839988-c19d4846-60f5-11e7-817c-b2a7e5978a54.png)

@xmo-odoo I believe this has to do with your unicode rewrites.

Regards,
Yenthe

**Comment by xmo-odoo** [comment]
It's fixed in saas-16: 98602e93cc3f8586db5d22932aecdd5192584a55

It probably hasn't been forward-ported yet.

@Yenthe666 technically that's not a unicode rewrite (that branch has yet to land), it's the API compatibility changes.
